src/lib/certmap/sss_cert_content_crypto.c

```diff
@@ -776,6 +776,35 @@ static int get_serial_number(TALLOC_CTX *mem_ctx, X509 *cert,
     return ret;
 }
 
+static int get_subject_key_id(TALLOC_CTX *mem_ctx, X509 *cert,
+                              uint8_t **subject_key_id,
+                              size_t *subject_key_id_size)
+{
+    const ASN1_OCTET_STRING *ski;
+    size_t size = 0;
+    uint8_t *buf;
+
+    ski = X509_get0_subject_key_id(cert);
+    if (ski != NULL) {
+        size = ASN1_STRING_length(ski);
+    }
+    if (size == 0) {
+        subject_key_id_size = 0;
+        subject_key_id = NULL;
```

Comment:
`*subject_key_id = NULL;`?

Reply:
I'm not sure what you mean here, but I think it is not necessary here to make a difference between unset and empty.

Reply:
I mean, to set output arg it should be dereferenced.